### ISSUE TITLE
Why do we  have LDAP/AD on Login Attributes tab when configuring ldap?

### ISSUE THREAD

**Issue by jnfrmarks** [owncloud/core#15948]
### Steps to reproduce

1. Install 8.1
2. Go to admin page and configure ldap
3. Click on Login Attributes tab

### Expected behaviour

No differentiation between LDAP and AD for the username or email address checkboxes

### Actual behaviour

We specifically call out LDAP/AD for the username and email address checkboxes

![p333](https://cloud.githubusercontent.com/assets/8480343/7398000/eed44eb8-ee79-11e4-9587-7df1fd24748b.PNG)

### Server configuration

ownCloud Enterprise Edition 8.1 alpha 2 (testing)

@blizzz @MTRichards

**Comment by DeepDiver1975** [comment]
@jnfrmarks sorry - but I don't get the meaning reason behind this issue. Do you ask why we name AD at all?

**Comment by jnfrmarks** [comment]
I was asking why we label it at all and if we are labeling it, why we specify both LDAP and AD.  Matt has indicated that was the ask.  I think it's a bit redundant but I kind of get why we're doing it :)  

I can close this one (I filed this and then chatted with Matt)

**Comment by DeepDiver1975** [comment]
> I can close this one (I filed this and then chatted with Matt)

with respect to the community: can we have the explanation in this ticket for everybody please? THX

**Comment by DeepDiver1975** [comment]
reopening until explanation has been entered - THX

**Comment by MorrisJobke** [comment]
@jnfrmarks Please explain it here :)

**Comment by blizzz** [comment]
> Rename the LDAP Username and LDAP Email Address lines to LDAP / AD Username and LDAP / AD Email Address

Is written in the feature catalogue (in S3 on Shared/owncloud/Product Management/LDAP config updates Jan 2015c.odt). Why, I can only assume, and that might be that AD users where irritated? I did not mind, so I did not ask.

@MTRichards and/or @bboule should know.

**Comment by DeepDiver1975** [comment]
> (in S3 on Shared/owncloud/Product Management/LDAP config updates Jan 2015c.odt)

is of zero use for the community

**Comment by blizzz** [comment]
Yes. But  I quoted the only relevant part about it.

**Comment by MTRichards** [comment]
A little explanation:
Technically we use LDAP for all communications with a directory (both AD and actual LDAP servers). Despite this, when we talk to admins and users (and the market) about the product we talk about LDAP and AD, because people have one or the other. Many administrators will not immediately recognize that LDAP = AD in this case.

LDAP / AD was specifically added to this page for this reason. The interesting point is that there is no LDAP / AD anywhere else in the wizard except this tab. In other tabs it is called LDAP - this was an oversight.

Bottom line: it needs to be made consistent across the wizard - where LDAP is replaced with LDAP / AD. In addition, since the app is called LDAP user and group backend, it probably also needs an update to LDAP / AD user and group backend in the description. This is much less confusing for all parties, and makes the suggestions consistent.

**Comment by DeepDiver1975** [comment]
Thx for the explanation!

**Comment by MorrisJobke** [comment]
> Bottom line: it needs to be made consistent across the wizard - where LDAP is replaced with LDAP / AD. In addition, since the app is called LDAP user and group backend, it probably also needs an update to LDAP / AD user and group backend in the description. This is much less confusing for all parties, and makes the suggestions consistent.

@nickvergessen @blizzz Want to fix this little thing? ;)
